components/admin_console/license_settings/modals/upload_license_modal.scss

```diff
@@ -0,0 +1,146 @@
+.UploadLicenseModal {
+    .modal-header {
+        border-bottom: none !important;
+        background: none !important;
+
+        button.close {
+            span {
+                color: rgba(63, 67, 80, 0.56) !important;
+            }
+        }
+    }
+
+    .modal-body {
+        padding-top: 0 !important;
+    }
+
+    .content-body {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        justify-content: center;
+
+        .svg-image {
+            margin-top: -20px;
+        }
+
+        .hands-svg {
+            margin-top: 0;
+            margin-bottom: 15px;
+        }
+
+        .title {
+            margin: 10px 0;
+            color: var(--sys-denim-center-channel-text);
```

Comment:
I don't think we need to use the theme colors for System Console, given that is not supported it may give inconsistent UX when they do have a theme selected